Zend/zend_execute.c

```diff
@@ -3239,7 +3239,7 @@ static zend_never_inline void zend_fetch_this_var(int type OPLINE_DC EXECUTE_DAT
 				Z_ADDREF_P(result);
 			} else {
 				ZVAL_NULL(result);
-				zend_error(E_WARNING, "Undefined variable: this");
+				zend_error(E_WARNING, "Undefined variable: $this");
```

Comment:
For the sake of consistency, drop `:` here too.

Reply:
No way I didn't see this.. Will fix